tests/debugger/bug01753.inc

```diff
@@ -1,5 +1,5 @@
-<?php
-require_once(__DIR__ . '/bug01753-SkeletonPage.inc');
-require_once(__DIR__ . '/bug01753-PageIndex.inc');
-$page = new PageIndex();
-$page->getData();
+<?php
+require_once(__DIR__ . '/bug01753-SkeletonPage.inc');
+require_once(__DIR__ . '/bug01753-PageIndex.inc');
+$page = new PageIndex();
+$page->getData();
```

Comment:
This seems an unrelated change, and hence don't add it to the commits in this PR.

Reply:
I opened https://github.com/xdebug/xdebug/pull/605 for it, I will then rebase this PR